lxd/config/map.go

```diff
@@ -94,8 +94,8 @@ func (m *Map) Dump() map[string]string {
 // GetRaw returns the value of the given key, which must be of type String.
 func (m *Map) GetRaw(name string) string {
 	value, ok := m.values[name]
-	// User key?
-	if shared.IsUserConfig(name) {
+	// User or dynamic storage key?
+	if shared.IsUserConfig(name) || shared.IsProjectStorageConfig(name) {
```

Comment:
IsUserConfig could be moved out of `shared/util.go` too (not a blocker ofc) because its not used by `client` package.

Reply:
or perhaps both can be moved in to `lxd/config/` package?